examples/resources/pingfederate_local_identity_identity_profile/resource.tf

```diff
@@ -1,6 +1,6 @@
-resource "pingfederate_local_identity_identity_profile" "localIdentityIdentityProfilesExample" {
+resource "pingfederate_local_identity_identity_profile" "myLocalIdentityIdentityProfile" {
   name = "yourIdentityProfileName"
-  id   = "yourid"
+  #id   = "yourid"
```

Comment:
Is this supposed to be uncommented, or deleted maybe?

Reply:
Oh I was testing something in that terraform file, good catch